README.md

```diff
@@ -19,8 +20,33 @@ Finally, use
 go build
 ```
 to fetch the latest dependencies.
+#### Install from binary (on Linux)
+Download the compressed binary file by running:
+```bash
+   curl -LJO https://github.com/adfolks/aksctl/releases/download/v0.1.0/aksctl-linux-amd64.tar.gz
```

Comment:
make this to https://github.com/adfolks/aksctl/releases/download/latest/aksctl-linux-amd64.tar.gz
version to latest